commpy/modulation.py

```diff
@@ -84,22 +88,64 @@ def demodulate(self, input_symbols, demod_type, noise_var = 0):
                     llr_den = 0
                     for const_index in self.symbol_mapping:
                         if (const_index >> bit_index) & 1:
-                            llr_num = llr_num + exp((-abs(current_symbol - self.constellation[const_index])**2)/noise_var)
+                            llr_num = llr_num + exp(
+                                (-abs(current_symbol - self.constellation[const_index]) ** 2) / noise_var)
                         else:
-                            llr_den = llr_den + exp((-abs(current_symbol - self.constellation[const_index])**2)/noise_var)
-                    demod_bits[i*self.num_bits_symbol + self.num_bits_symbol - 1 - bit_index] = log(llr_num/llr_den)
+                            llr_den = llr_den + exp(
+                                (-abs(current_symbol - self.constellation[const_index]) ** 2) / noise_var)
+                    demod_bits[i * self.num_bits_symbol + self.num_bits_symbol - 1 - bit_index] = log(llr_num / llr_den)
         else:
             pass
             # throw an error
 
         return demod_bits
 
+    def plotCons(self):
```

Comment:
@BastienTr @akou97 Same comment about naming conventions here. for e.g. plotCons would become plot_constellation.